src/sentry/eventstream/snuba.py

```diff
@@ -157,32 +161,37 @@ def insert(
 
         is_transaction_event = self._is_transaction_event(event)
 
+        extra_data: List[Mapping[Any, Any]] = [
+            {
+                "group_id": event.group_id,
+                "group_ids": [group.id for group in event.groups],
+                "event_id": event.event_id,
+                "organization_id": project.organization_id,
+                "project_id": event.project_id,
+                # TODO(mitsuhiko): We do not want to send this incorrect
+                # message but this is what snuba needs at the moment.
+                "message": event.search_message,
+                "platform": event.platform,
+                "datetime": event.datetime,
+                "data": event_data,
+                "primary_hash": primary_hash,
+                "retention_days": retention_days,
+            },
+            {
+                "is_new": is_new,
+                "is_regression": is_regression,
+                "is_new_group_environment": is_new_group_environment,
+                "skip_consume": skip_consume,
+            },
```

Comment:
What's the relationship between this and the new group states? Seems like we have some duplication now

Reply:
We'll consolidate this together later on. Ideally I'd like to remove the individual group properties and just rely on groups instead.

This is also just in the body and is currently unused since we use the headers

Reply:
Yeah, I'll open another PR to remove the redundant singular group params. Just wanted to slip in this multiple groups change and not make too many changes.

Reply:
Can you mark the parts that are supposed to be deprecated with comments or in the docstring then. Otherwise it can be hard to tell what direction this is supposed to be going in.